odxtools/scaleconstr.py

```diff
@@ -0,0 +1,50 @@
+# SPDX-License-Identifier: MIT
+from dataclasses import dataclass
+from enum import Enum
+from typing import Optional
+from xml.etree import ElementTree
+
+from .compumethods.limit import Limit
+from .exceptions import odxraise, odxrequire
+from .odxtypes import DataType
+from .utils import create_description_from_et
+
+
+class ValidType(Enum):
+    VALID = "VALID"
+    NOT_VALID = "NOT-VALID"
+    NOT_DEFINED = "NOT-DEFINED"
+    NOT_AVAILABLE = "NOT-AVAILABLE"
+
+
+@dataclass
+class ScaleConstr:
+    """This class represents a SCALE-CONSTR.
+    """
+
+    short_label: Optional[str]
```

Comment:
there is a short label, but no long label?

Reply:
yes, the XSD says only short label
![image](https://github.com/mercedes-benz/odxtools/assets/26408960/bbaf2afe-bdc3-4b08-9caf-4b653a094e31)